pkg/roachprod/promhelperclient/client.go

```diff
@@ -38,6 +40,18 @@ const (
 	ErrorMessagePrefix = "request failed with status %d"
 )
 
+// NodeProviderReachability is the reachability of the node provider.
+type Reachability int
+
+const (
+	// None indicates that the node provider is not reachable.
+	None Reachability = iota
+	// Private indicates that the node provider is reachable via private network.
+	Private
+	// Public indicates that the node provider is reachable via public network.
```

Comment:
To clarify, this just means the node is reachable *only* via public network right?

Reply:
From the perspective of Prometheus, yes.

Reply:
I clarified the comments.